packages/bot/src/components/icon-trade-types.jsx

```diff
@@ -31,39 +31,39 @@ import {
     TradeTypeUpOrDown,
     TradeTypeUnknown } from './Icons.jsx';
 
-const IconTradeType = ({ trade_type }) => {
+const IconTradeType = ({ trade_type, className }) => {
```

Comment:
Could we use `const IconTradeType = ({ trade_type, ...props }) => {` instead, to pass everything other than trade_type to the icons?

Reply:
@reza-fs why we need to pass everything?